composer.json

```diff
@@ -16,19 +16,15 @@
     }
   ],
   "require": {
-    "php": ">=7.2",
-    "contributte/di": "^0.5.0",
+    "php": ">=8.1",
```

Comment:
Can you please use PHP 8.2? And split commits to 1. codebase, 2. php version, 3. composer deps. Thank you. Also remove `v` from versions.

Reply:
Done